node-tests/acceptance/deliver-test.js

```diff
@@ -0,0 +1,127 @@
+const expect = require('chai').expect;
+const denodeify = require('denodeify');
+const request = denodeify(require('request'));
+const AddonTestApp = require('ember-cli-addon-tests').AddonTestApp;
+const fs = require('fs-extra');
+
+const cspMetaTagRegExp = /<meta http-equiv="Content-Security-Policy" content=".*">/i;
```

Comment:
Extremely minor (free free to keep it as is), but I think the JS convention is to use UPPER_CASE here:

```js
CSP_META_TAG_REG_EXP = /<meta http-equiv="Content-Security-Policy" content=".*">/i;
```